view/frontend/requirejs-config.js

```diff
@@ -1,5 +1,5 @@
-var config = {
-    map   : {
+const config = {
```

Comment:
@cammonro , I'm sure there is a good reason why you changed it to `const`. But it looks a little odd to me. 
With my limited FE knowledge, I believe `const` variable is used so that it can not be updated. However, all config should be merged together and create a global config. So, `var` instead of `const` make sense to me. 

Also, I looked at Magento vendor library, I couldn't find Magento is shifting towards `const`.